src/main/java/io/jenkins/plugins/checks/github/GitHubChecksPublisher.java

```diff
@@ -54,10 +57,12 @@ public void publish(final ChecksDetails details) {
                     credentials);
             GHCheckRunBuilder builder = createBuilder(gitHub, new GitHubChecksDetails(details));
             builder.create();
+            listener.getLogger().println("GitHub checks have been published.");
         }
         catch (IllegalStateException | IOException e) {
-            //TODO: log to the build console
-            LOGGER.log(Level.WARN, "Could not publish GitHub check run", e);
+            String message = "Failed Publishing GitHub checks: " + e;
+            LOGGER.log(Level.WARN, message);
```

Comment:
always log the exception with the throwable arg
```suggestion
            LOGGER.log(Level.WARN, message, e);
```